nh_observations/notifications.py

```diff
@@ -129,6 +129,11 @@ def complete(self, cr, uid, activity_id, context=None):
         }
     ]
 
+    def set_form_description_frequencies(self, available_frequencies):
```

Comment:
I think the method should still work okay even if `_form_description` is overridden lower down because it's being called from lower down with `self`. This method is only used in my code in nh_configurations currently but seemed best to put it here for reuse as it could be used anytime with a *nh.clinical.notification.frequency* object.